From ac4c0bedea8de60a8f0a4cbb231ea7fb47a55e35 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 18 May 2017 14:43:56 +0300
Subject: [PATCH 54/78] ARM: i.MX6: dts: cl-som-imx6 add mipi to fb0 mapping

Add mipi to fb0 mapping, that allows using a mipi display as
the device default endpoint.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6q-sbc-imx6-mipi.dts  | 15 +++++++++++++++
 arch/arm/boot/dts/imx6qp-sbc-imx6-mipi.dts | 15 +++++++++++++++
 2 files changed, 30 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6q-sbc-imx6-mipi.dts
 create mode 100644 arch/arm/boot/dts/imx6qp-sbc-imx6-mipi.dts

diff --git a/arch/arm/boot/dts/imx6q-sbc-imx6-mipi.dts b/arch/arm/boot/dts/imx6q-sbc-imx6-mipi.dts
new file mode 100644
index 0000000..ba5dfeb
--- /dev/null
+++ b/arch/arm/boot/dts/imx6q-sbc-imx6-mipi.dts
@@ -0,0 +1,15 @@
+/*
+* Copyright 2016 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+#include "imx6q-sbc-imx6.dts"
+#include "imx6qdl-cm-fx6-mipi.dtsi"
diff --git a/arch/arm/boot/dts/imx6qp-sbc-imx6-mipi.dts b/arch/arm/boot/dts/imx6qp-sbc-imx6-mipi.dts
new file mode 100644
index 0000000..00c95b8
--- /dev/null
+++ b/arch/arm/boot/dts/imx6qp-sbc-imx6-mipi.dts
@@ -0,0 +1,15 @@
+/*
+* Copyright 2016 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+#include "imx6qp-sbc-imx6.dts"
+#include "imx6qdl-cm-fx6-mipi.dtsi"
-- 
1.9.1

